libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -1124,18 +1112,18 @@ class MapRouteLineTest {
             "[\"rgba\", 0.0, 0.0, 0.0, 0.0], 0.01792979, [\"rgba\", 47.0, 122.0, 198.0, 1.0]]"
         val expectedRouteTrafficLineVanishingExpression = "[\"step\", [\"line-progress\"], " +
             "[\"rgba\", 0.0, 0.0, 0.0, 0.0], 0.01792979, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.056129422, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.09373502, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.150941, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.1905395, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.26964808, " +
-            "[\"rgba\", 255.0, 77.0, 77.0, 1.0], 0.27944908, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.33272266, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.39298487, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.48991048, " +
-            "[\"rgba\", 255.0, 77.0, 77.0, 1.0], 0.504132, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.8017851, " +
-            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 1.0000086, [\"rgba\", 86.0, 168.0, 251.0, 1.0]]"
+            "[\"rgba\", 86.0, 168.0, 251.0, 1.0], 0.05612764, " +
```

Comment:
Why did the offset values change in the tests? They don't seem to match any previous values from the array that would indicate that we simply skipped some entries.

Reply:
I misunderstood what you were asking about. Look at the tests getRouteLineTrafficExpressionDataWhenStreetClassDataExists and getRouteLineTrafficExpressionDataWhenStreetClassDoesNotExist.

There was a slight change in the way the distances in the route were added up but from what I looked at the difference was within 4 or 5 decimal places. With such a small degree of difference I don't think it's erroneous because it's a tiny fraction of a metere.

Reply:
Oh, so what you're saying is that turf calculations were different than what Directions API returns? If that's the case, I guess it makes us only more precise.